apps/sms/test/marionette/mocks/mock_message_thread.js

```diff
@@ -0,0 +1,38 @@
+module.exports = {
```

Comment:
I don't feel good about this mock... It feels like it does too much of unrelated things: inject dependencies, create mock thread instances. I'd rather have in tests to be more expressive and make test more readable, but it's up to you to have such mock somewhere in shared tests :)

We have something like mock_threads in unit tests and I still have hard time re-using/reading it :)